doc/source/release-notes/Percona-Server-8.0.13-4.rst

```diff
@@ -0,0 +1,50 @@
+.. rn:: 8.0.13-4
+
+================================================================================
+|Percona Server| |release|
+================================================================================
+
+|Percona| announces the release of |Percona Server| |release| on
+|date| (downloads are available `here
+<https://www.percona.com/downloads/Percona-Server-8.0/>`__ and from
+the `Percona Software Repositories
+<https://www.percona.com/doc/percona-server/8.0/installation.html#installing-from-binaries>`__).
+This release contains a fix for a critical bug that prevented |Percona
+Server| 5.7.24-26 from being upgraded to version 8.0.13-3. |Percona
+Server| |release| is now the current GA release in the 8.0 series. All
+of |Percona|’s software is open-source and free.
+
+Percona Server for MySQL 8.0 includes all the `features available in MySQL 8.0
+Community Edition
+<https://dev.mysql.com/doc/refman/8.0/en/mysql-nutshell.html>`__ in addition to
+enterprise-grade features developed by Percona.  For a list of
+highlighted features from both MySQL 8.0 and Percona Server for MySQL 8.0,
+please see the `GA release announcement
+<https://www.percona.com/blog/2018/12/21/announcing-general-availability-of-percona-server-for-mysql-8-0/>`__.
+
+.. note::
+
+   If you are upgrading from 5.7 to 8.0, please ensure that you read the
+   `upgrade guide
+   <https://www.percona.com/doc/percona-server/8.0/upgrading_guide.html>`__ and the
+   document `Changed in Percona Server for MySQL 8.0
+   <https://www.percona.com/doc/percona-server/8.0/changed_in_version.html>`__.
+
+
+Bugs Fixed
+================================================================================
+
+- It was not possible to upgrade from MySQL 5.7.24-26 to 8.0.13-3. Bug fixed :psbug:`5245`.
+- ``SHOW BINLOG EVENTS FROM <bad offset>`` is not diagnosed inside ``Format_description_log_events``. Bug fixed :psbug:`5126` (Upstream :mysqlbug:`93544`).
+- ``innodb.upgrade_orphan`` failed because of left files. Bug fixed :psbug:`5209` (Upstream :mysqlbug:`93686`).
+- There was a typo in `mysqld_safe.sh`: **trottling** was replaced with **throttling**. Bug fixed :psbug:`240`.
+- `main.mysqldump` failed because of dropped event. Bug fixed :psbug:`5268` (Upstream :mysqlbug:`93788`).
+- ``EXPLAIN SELECT`` returned inconsistent number of ROWS in ``main.group_by``. Bug fixed :psbug:`5306` (Upstream :mysqlbug:`93703`).
+- On Debian or Ubuntu, a conflict was reported on the
+  :file:`/usr/bin/innochecksum` file when attempting to install |Percona Server| 8
+  over the |MySQL| 8. Bug fixed :psbug:`5225`.
+
+Other bugs fixed: :psbug:`4797`:, :psbug:`5264`:, :psbug:`5270`:, :psbug:`5309`:, :psbug:`5311`:
```

Comment:
5311: an out-of-bound read may occur on debug builds in the compressed columns with dictionaries feature